src/Kartlag/AktiveKartlag/EkspandertMeny/FellesElementer/VelgFargeBoks.js

```diff
@@ -7,6 +7,7 @@ class VelgFargeBoks extends React.Component {
       <div
         className="colour_legend"
         title={"Velg farge"}
+        aria-label={"Velg farge"}
```

Comment:
Mistenker denne også er en plass de vil vite hva fargen skal velges for, siden de ikke kan.. se...
men om de ikke kan se, trenger de da farge? jeg er forvirra

Reply:
Det kan jo være sånn at de ser, men litt dårligere enn de fleste, og trenger da sterkere farger for å se bedre?

Reply:
Godt poeng! Vet vi hvilken det er? har vi tilgang til elementet? (antar vi har det, men mulig det må sendes inn ovenfra)

Reply:
Navnet på kartlaget ligger ikke i klassen (hvis jeg har forstått spørsmålet riktig), men tror det burde gå an å sende det til fargeboksvelger.